_data/projects/simon-quality.yml

```diff
@@ -0,0 +1,11 @@
+---
+name: Air and Ground Monitoring Information System using Arduino UNO R3
+desc: Sistem Informasi Monitoring Udara dan Tanah menggunakan Arduino UNO R3
+site: https://bellabeen.github.io
+tags:
+  - arduino
+  - iot
+
+upforgrabs:
+  name: Sistem Informasi Monitoring Udara dan Tanah menggunakan Arduino UNO R3
+  link: https://github.com/bellabeen/simon_quality
```

Comment:
@bellabeen This link should be one of the labels that you use to mark your issues. you have several options for your Repository at https://github.com/bellabeen/simon_quality/labels.

You can use any of the such as https://github.com/bellabeen/simon_quality/labels/good%20first%20issue.

Suppose you are using `good first issue` Label to mark your issues, then you need to set the `name` at line 11 in this file as `good first issue` and `link` at line 12 as `https://github.com/bellabeen/simon_quality/labels/good%20first%20issue`.

Also, once you decide the label. You need to open some issues at your repository so that other contributors comming to you project can contribute easily by picking up those issues.

Reply:
Oke thanks, i resolve this problem